arubabank/__init__.py

```diff
@@ -74,9 +78,9 @@ def login(self, username, password):
             sys.exit()
         return login_response
 
-    def get_portfolios(self):
+    def get_portfolio_ids(self):
```

Comment:
With the get_portfolios(), we get all the portfolio data: id, name, etc... And then somewhere in the main program we can extract only the ids when we need them, that way we only make 1 api call to get all portfolio data. 

With this specific api call to get only portfolio_ids we would then have to make another api call to get portfolio names if we need them. So I say leave the get_portfolios() as-is and then make the get_porfolio_ids() get the ids from the get_portfolios() function.

Thoughts?